### ISSUE TITLE
LSP Support

### ISSUE THREAD

**Issue by danillos** [out-of-cheese-error/the-way#151]
Would be good to have LSP support built-in. So we could use it inside editors.

**Comment by Ninjani** [comment]
That would be a nice feature. Do you have some examples of libraries where this is implemented?

**Comment by danillos** [comment]
Yes, I'm using this one https://github.com/Stanislav-Lapata/snippets-ls on Helix Editor.
